docs/REQUIREMENTS.md

```diff
@@ -7,8 +7,197 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Definitions
 
-- _**Term**: Definition._
+- **Image Compatibility specification**: A specification that describes the container image requirements for the host operating system.
```

Comment:
```suggestion
- **Image Compatibility specification**: A specification that describes container image requirements for a host operating system.
```
Suggestion to make a little more generic, "the" implies a specific host (ours) and theoretically it's for _a_ host.

Reply:
If we use "spec" below we might say "A specification (spec)..." up here.